src/main/kotlin/no/nav/api/pdl/PdlClient.kt

```diff
@@ -1,54 +1,63 @@
 package no.nav.api.pdl
 
-import HentAktorid
+import com.expediagroup.graphql.client.types.GraphQLClientResponse
+import io.ktor.client.*
 import io.ktor.client.engine.*
 import io.ktor.client.engine.okhttp.*
 import io.ktor.client.request.*
-import no.nav.api.pdl.queries.HentNavn
-import no.nav.api.pdl.queries.HentPersonalia
+import no.nav.api.generated.pdl.HentPersonalia
+import no.nav.api.generated.pdl.HentAktorid
+import no.nav.api.generated.pdl.HentNavn
 import no.nav.utils.*
+import java.net.URL
 
 class PdlClient(
     private val pdlUrl: String,
     private val tokenclient: BoundedMachineToMachineTokenClient,
     httpEngine: HttpClientEngine = OkHttp.create()
 ) {
-    private val graphqlClient = GraphQLClient(
-        httpClient = GraphQLClient.createHttpClient(httpEngine),
-        config = GraphQLClientConfig(
-            serviceName = "PDL",
-            critical = false,
-            requestConfig = {
-                val token = tokenclient.createMachineToMachineToken()
-                url(pdlUrl)
-                header("Nav-Consumer-Token", "Bearer $token")
-                header("Authorization", "Bearer $token")
-                header("Tema", "GEN")
-                header("X-Correlation-ID", getCallId())
-            }
-        )
+    private val graphqlKtorClient = LoggingGraphQLKtorClient(
```

Comment:
"ktor" i variabelnavnet gir oss egentlig ikke noe nyttig info, så tenker vi egentlig kan holde navnene litt enklere. 
```suggestion
private val graphqlClient = LoggingGraphQLKtorClient(
```